### PR TITLE
Update bookinfo sample images (#10552)

### PR DESCRIPTION
**Pull request by rvennam**
- Minor changes to `build_push_update_images.sh` script
  - fix to make it work on OS X
  - fix bug with IMAGES list
- Ran `build_push_update_images.sh 1.10.0` which:
  - builds all bookinfo images and tags with 1.10.0 and latest
  - pushes images to docker hub
  - updates yaml files 1.9.0 -> 1.10.0

### CODE REVIEW

**Comment by rvennam** [comment]
fixes #10552

File: samples/bookinfo/build_push_update_images.sh [hunk 1]
```diff
@@ -18,18 +18,28 @@ set -o errexit
...
-sed -i "s/\\(istio\\/examples-bookinfo-.*\\):[[:digit:]]\\.[[:digit:]]\\.[[:digit:]]/\\1:$VERSION/g" -- */bookinfo*.yaml
+#push images
+for IMAGE in ${IMAGES}; 
+do 
+  echo "Pushing: ${IMAGE}" 
+  docker push "${IMAGE}"; 
+done
 
+#Update image references in the yaml files
+find . -name "*bookinfo*.yaml" -exec sed -i.bak "s/\\(istio\\/examples-bookinfo-.*\\):[[:digit:]]\\.[[:digit:]]\\.[[:digit:]]/\\1:$VERSION/g" {} +
```
Comment (vadimeisenbergibm) [inline samples/bookinfo/build_push_update_images.sh]: Missing newline character

File: samples/bookinfo/src/reviews/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.gradle
```
Comment (vadimeisenbergibm) [inline samples/bookinfo/src/reviews/.gitignore]: missing newline character

File: samples/bookinfo/src/reviews/reviews-application/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/build/
```
Comment (vadimeisenbergibm) [inline samples/bookinfo/src/reviews/reviews-application/.gitignore]: missing newline character

**Comment by linsun** [comment]
opened an issue for the flaky test https://github.com/istio/istio/issues/10706

/lgtm

**Pull request merged**